src/test/java/ProductPageTests.java

```diff
@@ -0,0 +1,28 @@
+import hwa_18.HomePage;
+import hwa_18.ProductDetailPage;
+import hwa_18.SearchResultPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ProductPageTests extends BaseTest{
+
+    @Test
+    public void verifyProductDetailTitle() throws InterruptedException {
```

Comment:
Do we need throws InterruptedException here?

Reply:
fixed